sopel/loader.py

```diff
@@ -139,7 +139,7 @@ def is_triggerable(obj):
 
     A triggerable is a callable that will be used by the bot to handle a
     particular trigger (i.e. an IRC message): it can be a regex rule, an
-    event, an intent, a command, a nickname command, or an action command.
+    event, a CTCP command, a command, a nickname command, or an action command.
     However, it must not be a job or a URL callback.
```

Comment:
Would you mind rewrapping this paragraph? Can be its own commit if you want; it just annoys me now to have one line at 79c when most are 72c or less.